boards/samr30-xpro/Makefile.dep

```diff
@@ -5,3 +5,7 @@ endif
 ifneq (,$(filter saul_default,$(USEMODULE)))
   USEMODULE += saul_gpio
 endif
+
+ifneq (,$(filter ztimer_msec,$(USEMODULE)))
+  USEMODULE += ztimer_periph_rtt
+endif
```

Comment:
this one can get in by simply dropping this change @jue89, care to rebase?

Reply:
I can't remember ... what was the problem with this change?

Reply:
Since a while `periph_rtt` is the default `ztimer_msec` backend, except for some BOARDs, among them sam0 BOARDs since it was shown that the rtt overhead was to high to be able to implement a good enough `msec` timer  by default (see https://github.com/RIOT-OS/RIOT/pull/17395):

https://github.com/RIOT-OS/RIOT/blob/f298d0a5853f690a9ab86421165525e0d52fe655/sys/ztimer/Makefile.dep#L81-L90

I I just ran `USEMODULE=ztimer_periph_rtt IOTLAB_NODE=samr30-1.saclay.iot-lab.info BOARD=samr30-xpro make -C tests/bench_ztimer/` for samr30 and it looks OK:

```
main(): This is RIOT! (Version: 2022.04-devel-798-g8fa26-HEAD)
ztimer benchmark application.

                     set() one   121995 / 1000 = 121
                  remove() one     1544 / 1000 = 1
          set() + remove() one   243996 / 1000 = 243
  set() many increasing target   138007 / 1000 = 138
               re-set()  first   122001 / 1000 = 122
               re-set() middle   122100 / 1000 = 122
               re-set()   last   122192 / 1000 = 122
       remove() + set()  first   244009 / 1000 = 244
       remove() + set() middle   243983 / 1000 = 243
       remove() + set()   last   243982 / 1000 = 243
      remove() many decreasing   203144 / 1000 = 203
                  ztimer_now()     7506 / 1000 = 7
              sizeof(ztimer_t)    16000 / 1000 = 16
done.
```

So it looks like the `sam%` blacklist was a bit too strict and we can remove `sam3`.

Reply:
Well, that patch completely breaks our application. We need `ZTIMER_MSEC` running on `periph_rtt`. Yes, high `RTT_MIN_OFFSETS` are required, but with that in place, the application just works fine even with all these busy loops. We don't have another choice for battery applications.

Reply:
Note that `ztimer_no_periph_rtt` does not impede including `periph_rtt` it just does not enable it by default. So I don't see why that would break your application.

Reply:
I also ran `RTT_MIN_OFFSET` for the BOARD and it yiels:

```
RTT_MIN_OFFSET for samr30-xpro over 1024 samples: 8
```

So quite OK, I think we should re-enable it.

Reply:
> So quite OK, I think we should re-enable it.

Ah wait this is already configured

Reply:
> So I don't see why that would break your application.

That line made me saying this ;-)
https://github.com/RIOT-OS/RIOT/blob/df8086091f20a72d3e1380376684df39ae922d13/sys/ztimer/Makefile.dep#L88-L90

Reply:
Yes but it only prevents "auto-inclusion", it doesn't forbid an application from till requesting, so you can have:

`USEMODULE += ztimer_no_periph_rtt ztimer_periph_rtt`.

I'll open a PR re-enabling `sam3`